setup.py

```diff
@@ -332,7 +332,7 @@ def run(self):
 
 # Runtime requirements.
 inst_reqs = [
-    'affine', 'attrs', 'cligj>=0.5', 'numpy', 'snuggs>=1.4.1', 'click-plugins']
+    'affine', 'attrs', 'cligj>=0.5', 'numpy', 'snuggs>=1.4.1', 'click==7.0', 'click-plugins']
```

Comment:
Requirements shouldn't be pinned this tightly in a library's setup.py as it can make application dependencies too hard to satisfy. In cligj 0.5.0 we have `'click >= 4.0, < 8'`. We're counting on this to pull in click, but I think we should also spec `'click >= 4.0, < 8'` here in Rasterio.

Reply:
> @ocefpaf requirements shouldn't be pinned this tightly in a library's setup.py as it can make application dependencies too hard to satisfy.

I agree, but I was following the lead on `requirements.txt` here. Anyway, now I see that your use for `requirements.txt` is quite different from what I am used.